cache/redis_internal_test.go

```diff
@@ -45,7 +45,29 @@ func TestNewRedis(t *testing.T) {
 	c := v.(*redisCache)
 
 	assert.NoError(t, err)
-	assert.Equal(t, 12, c.client.Options().PoolSize)
+	clusterClient, isCluster := c.client.(*redis.ClusterClient)
+	if isCluster {
+		assert.Equal(t, 12, clusterClient.Options().PoolSize)
+	} else {
+		client, ok := c.client.(*redis.Client)
+		assert.True(t, ok)
+		assert.Equal(t, 12, client.Options().PoolSize)
+	}
+}
+
+func TestNewRedisUniversal(t *testing.T) {
+	v, err := NewRedisUniversal([]string{"redis://test", "redis://test2"}, WithPoolSize(12))
+	c := v.(*redisCache)
+
+	assert.NoError(t, err)
+	clusterClient, isCluster := c.client.(*redis.ClusterClient)
```

Comment:
Rather assert that it is indeed a cluster, as we expect it to be.

Reply:
ok